backend/src/main/resources/db/migration/local/V3_1__LevelsOperations.sql

```diff
@@ -1,15 +1,26 @@
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Beginner', 0, 20, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Advanced Beginner', 21, 60,'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Professional', 61, 99,'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (2, 'Junior', 0, 30, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (2, 'Advanced Beginner', 31, 70, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (2, 'Intermediate', 71, 99, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (3, 'Beginner', 0, 25, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (3, 'Advanced Beginner', 26, 45, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (3, 'Professional', 46, 99, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (4, 'Beginner', 0, 40, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (4, 'Mid Level', 41, 80, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (4, 'Intermediate', 81, 99, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (5, 'Beginner', 0, 10, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (5, 'Advanced Beginner', 11, 50, 'https://roadmap.sh/');
-INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (5, 'Intermediate', 51, 99, 'https://roadmap.sh/');
+INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Beginner', 0, 20, 'https://roadmap.sh/ai/backend-developer');
+INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Advanced Beginner', 21, 40, 'https://roadmap.sh/ai/from-beginner-to-advanced-beginner-backend-developer');
+INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Intermediate', 41, 60, 'https://roadmap.sh/ai/from-advanced-beginner-to-intermediate-backend-developer');
+INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Proficient', 61, 80, 'https://roadmap.sh/ai/from-intermediate-to-proficient-backend-developer');
+INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Expert', 81, 99, 'https://roadmap.sh/ai/from-proficient-to-expert-backend-developer');
```

Comment:
```suggestion
INSERT INTO levels (role_id, name, min_coefficient, max_coefficient, link) VALUES (1, 'Expert', 81, 100, 'https://roadmap.sh/ai/from-proficient-to-expert-backend-developer');
```

We had a problem putting the maximum value as 99, no?